janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java

```diff
@@ -356,14 +358,17 @@ public HBaseStoreManager(org.janusgraph.diskstorage.configuration.Configuration
 
     @Override
     public Deployment getDeployment() {
-        List<KeyRange> local;
-        try {
-            local = getLocalKeyPartition();
-            return null != local && !local.isEmpty() ? Deployment.LOCAL : Deployment.REMOTE;
-        } catch (BackendException e) {
-            // propagating StorageException might be a better approach
-            throw new RuntimeException(e);
+        if (deployment == null) {
```

Comment:
It appears the style guide here is to write `null == deployment` to avoid a typo of the form:

```java
if (deployment = null) {
    ...
}
```